justfile

```diff
@@ -0,0 +1,234 @@
+# justfile for Rnote
+
+# Either 'true' or 'false'
+ci := "false"
+linux_distr := `lsb_release -ds | tr '[:upper:]' '[:lower:]'`
+log_level := "debug"
+build_folder := "_mesonbuild"
+flatpak_app_folder := "_flatpak_app"
+flatpak_repo_folder := "_flatpak_repo"
+mingw64_prefix_path := "C:/msys64/mingw64"
+
+sudo_cmd := if ci == "true" {
+    ""
+} else {
+    "sudo"
+}
```

Comment:
Isn't running sudo in GH Actions just a no-op, or is this necessary?

Reply:
Wouldn't it be dependent on the used job container image? This originally came from a gitlab workflow. I am not sure if it's needed. I'll keep the `sudo_cmd` variable, but assign it unconditionally to `sudo`.

Reply:
That's true, I was assuming that it's root. If it's not root, then we'd need sudo anyways. This should always work as long as sudo is pre-installed (or there's some kind of shim) in the image.